lib/logstash/inputs/lumberjack.rb

```diff
@@ -48,7 +51,7 @@ def register
 
     @logger.info("Starting lumberjack input listener", :address => "#{@host}:#{@port}")
     @lumberjack = Lumberjack::Server.new(:address => @host, :port => @port,
```

Comment:
@urso even if the test is done at the library level its better to add validations inside the plugin and raise a `LogStash::ConfigurationError`, its bit more user friendly. We hope to provide finer grained validation options for the attributes in the future.